egs2/voxceleb/spk1/conf/tuning/train_RawNet3_sgdr_bs512_da_sampler_adam.yaml

```diff
@@ -35,28 +40,34 @@ loss_conf:
   margin: 0.3
   scale: 30
 
-max_epoch: 40
+max_epoch: 160
 #num_iters_per_epoch: 1
 num_att_plot: 0
 num_workers: 8
 cudnn_deterministic: False
 cudnn_benchmark: True
-batch_size: 128
-#batch_type: unsorted
-iterator_type: sequence
-shuffle_within_batch: True
-log_interval: 50
-optim: adamw
+iterator_type: category
+valid_iterator_type: sequence
+shuffle_within_batch: False
+log_interval: 100
+batch_size: 512
+valid_batch_size: 40
+optim: adam
 optim_conf:
   lr: 0.001
   weight_decay: 0.00005
   amsgrad: False
 
 scheduler: CosineAnnealingWarmupRestarts
 scheduler_conf:
-  first_cycle_steps: 77544
+  first_cycle_steps: 73024
```

Comment:
How do you determine this number? Is there some strategy to calculate it if we want to use the configuration for another dataset?

Reply:
Good question! Currently it's `n_batch * 32`. But actually this value can be empirically designed, even handpicked values such as 75000 would be okay. 
I think it would be difficult to give a guideline for this one..